packages/editor-ui/src/stores/nodeCreator.ts

```diff
@@ -9,4 +195,110 @@ export const useNodeCreatorStore = defineStore(STORES.NODE_CREATOR, {
 		showScrim: false,
 		selectedType: ALL_NODE_FILTER,
 	}),
+	actions: {
+		setShowTabs(isVisible: boolean) {
+			this.showTabs = isVisible;
+		},
+		setShowScrim(isVisible: boolean) {
+			this.showScrim = isVisible;
+		},
+		setSelectedType(selectedNodeType: INodeFilterType) {
+			this.selectedType = selectedNodeType;
+		},
+		setFilter(search: string) {
+			this.itemsFilter = search;
+		},
+		setAddedNodeActionParameters (action: IUpdateInformation, telemetry?: Telemetry, track = true) {
+			const { $onAction: onWorkflowStoreAction } = useWorkflowsStore();
```

Comment:
Nice! Didn't know you can subscribe to particular action using pinia. Will add that to pinia doc.

Reply:
Yup! I ❤️ Pinia